src/realtime/flood/flood_hazard.py

```diff
@@ -149,23 +161,32 @@ def write_keywords(self):
         }
 
         keywords = {
-            'hazard_category': hazard_category_single_event['key'],
+            hazard_category['key']: hazard_category_single_event['key'],
             'title': self.localization.hazard_title.format(
                 timestamp=self.event_time_in_time_zone),
-            'keyword_version': inasafe_keyword_version,
+            inasafe_keyword_version_key: inasafe_keyword_version,
             'value_maps': {
                 exposure_structure['key']: hazard_classes,
                 exposure_place['key']:  hazard_classes,
                 exposure_land_cover['key']:  hazard_classes,
                 exposure_road['key']:  hazard_classes,
                 exposure_population['key']:  hazard_classes,
             },
-            'hazard': hazard_flood['key'],
+            layer_purpose['key']: layer_purpose_hazard['key'],
+            layer_purpose_hazard['key']: hazard_flood['key'],
             'source': self.data_source.source_name(),
-            'layer_purpose': layer_purpose_hazard['key'],
-            'layer_geometry': layer_geometry_polygon['key'],
+            layer_geometry['key']: layer_geometry_polygon['key'],
             'inasafe_fields': {hazard_value_field['key']: u'state'},
-            'layer_mode': layer_mode_classified['key']
+            layer_mode['key']: layer_mode_classified['key'],
+
+            # Extra keywords
```

Comment:
Extra keys for @myarjunar to review.

Reply:
For @ismailsunni : I included generic extra_keyword_time_zone, will it clash with Desktop? Maybe in Wizard or so?

Reply:
@lucernae I think it won't be shown in the wizard. You need to register it in the flood extra keywords: https://github.com/inasafe/inasafe/blob/develop/safe/definitions/extra_keywords.py#L416

But I think it will run fine (can be accessed by reading the keywords)

Reply:
LGTM thanks @lucernae